benchmarks/doneahuerta/doneahuerta.cc

```diff
@@ -0,0 +1,375 @@
+#include <aspect/simulator.h>
```

Comment:
can you copy the usual copyright header here?

Reply:
done